### ISSUE TITLE
Review of updated HowTo examples

### ISSUE THREAD

**Issue by RichardWallis** [schemaorg/schemaorg#1899]
Following commit of PR #1895, the updated examples need reviewing.
- In [comment](https://github.com/schemaorg/schemaorg/pull/1895#issuecomment-384793938) @danbri raised a concern that the steps section is broken. 
- @dbiollo identified that the name of the HowToTool will not be set with this syntax.

**Comment by RichardWallis** [comment]
There are two approaches to getting the name of the HowToTool, and HowToSuppy, to be picked out.
```
<div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">Spare tire</div>
```
or
```
<div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">
     <span itemprop="name">Spare tire</span>
</div>
```
My preference is for the second of these as it could be expanded thus:
```
<div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">
     <span itemprop="name">Spare tire</span>
     <img itemprop="image" src="spare-tire.jpg" />
</div>
```

**Comment by dbiollo** [comment]
Maybe makes sense to show both ways in the sample? From a unscientific survey of "howtos" on various web sites, it looks like plain text tools is more common, but there are definitely a few that also show images.

If I had the job of marking up pages, I'd tend to prefer the simplest possible markup :)

**Comment by RichardWallis** [comment]
Released in V3.4
